docs/technical-overview/architecture.md

```diff
@@ -22,6 +22,22 @@ We also use server-side caching: [Rails caching][rails_caching]. If you see
 `Rails.cache` or `<%= cache ... %>`, this is code affected in production by
 caching.
 
+## Content precision
+
+In some situations we may want more precise content than in others. Often when
+we do not need a precise number, it offers an opportunity to either estimate
+the count or bust the cache less frequently.
```

Comment:
```suggestion
the content or bust the cache less frequently.
```
Or, if you want to keep count here, maybe change line 25 to read "Count precision"? 🤔 Right now, it sounds like this should apply to things beyond just estimating count.